skills/imported/autoreview/tests/test_autoreview_hardening.py

```
@@ -246,6 +262,483 @@ def installed_java() -> str | None:
+                    "overall_explanation": "Synthetic clean.", "overall_confidence": 0.9,
+                }),
+            }), contextlib.redirect_stdout(io.StringIO()), contextlib.redirect_stderr(io.StringIO()):
+                passes = self.helper["prepare_review_prompts"](repo, "local", None, captured, "", evidence, 30_000)
+                self.assertGreater(len(passes), 1)
+                self.assertEqual(len(scans), 1)
+                self.assertIn("SOURCE_ONLY_SCAN_SENTINEL\nMULTILINE_SCAN_CONTINUATION\n", scans[0])
+                self.assertIn(evidence[0].content, scans[0])
+                args = argparse.Namespace(engine="codex", max_priority="P0")
+                self.helper["run_review_passes"](args, [args], repo, passes, captured, False)
+            self.assertEqual(scans[1:], sends)
+            for item, scanned in zip(passes, scans[1:]):
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)